src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCounter = 0;
+
+  return function getSum(num1) {
+    return function(num2) {
+      callCounter++;
+
+      if (callCounter >= 4 && callCounter % 2 === 0) {
```

Comment:
```suggestion
      if (callCounter > 3 && callCounter % 2 === 0) {
```
it looks more obvious